src/main/java/com/realestate/service/user/entity/User.java

```diff
@@ -9,21 +9,25 @@
 import javax.persistence.Id;
 
 import com.realestate.service.common.entity.BaseDateTimeEntity;
+
 import com.realestate.service.user.constant.Role;
```

Comment:
```suggestion
import com.realestate.service.common.entity.BaseDateTimeEntity;
import com.realestate.service.user.constant.Role;
```

체크스타일 ```ide```에서는 안걸리는데  ```./gradlew checkstyleMain``` 명령어로 돌렸을때 
체크스타일 하나가 걸려서 확인해 보니까 ```8.25```버전에 이슈가 하나있더라고요ㅠㅠ
그래서 버전을 ```8.28```로 올리면 다행이 이슈가 해결된 버전이여서 유효하게 나타나네요...ㅎㅎ
수정 부탁드리겠씁니다 :)

![image](https://user-images.githubusercontent.com/47877273/153063184-33b64b64-45d9-455b-a2f1-5a906d088617.png)

![image](https://user-images.githubusercontent.com/47877273/153063630-737a700f-d69d-4af5-94f6-61f28a9d6261.png)